Library/Homebrew/tap.rb

```diff
@@ -209,6 +209,7 @@ def official?
   # True if the remote of this {Tap} is a private repository.
   def private?
     return @private if instance_variable_defined?(:@private)
+    return false unless installed?
```

Comment:
What's this needed for?

Reply:
Analytics are not reported for private repos.
https://github.com/Homebrew/brew/blob/9fb771d2a4d9d6a3b0677abf6915b06a2767cddf/Library/Homebrew/formula_installer.rb#L408-L413

Reply:
Yes but why does this need added here? I'm not sure I understand how this relates to this PR.

Reply:
When the `brewsci/bio` tap is not tapped (cloned), without this line, it fails with…
```console
$ brew install ghcr.io/brewsci/bio/seqkit
==> Downloading https://ghcr.io/v2/brewsci/bio/seqkit/blobs/sha256:831fc5b89602a
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/brewsci/bio/seqkit/blobs/sha256:831fc5b89602a
Already downloaded: /Users/shaun.jackman/Library/Caches/Homebrew/downloads/fdcdf45495262153d2a02046cf078be86615b3da02f5696ccf08c16413900dea--sha256:831fc5b89602adc8cfaa943c40522a345a020d3487b5d06b89d3aaf36a9e408b
==> Pouring the cached bottle
==> Installing seqkit from brewsci/bio
Error: No available tap brewsci/bio.
```

Reply:
What's the backtrace for that (found with `--debug`)? I think I'd rather fix the call site here.